Billwilke42/ideabox-boilerplate
# Feature/display cards

**What’s this PR do?**
Add functionality to display cards, deletes hard coded cards, add functionality to disable save button
**Where should the reviewer start?**
main.js : line 20
styles.css : darkness-disabled
**How should this be manually tested?**
Note that click button is disable on load
Add text to title and body inputs and click save button
